pdsc/ingest.py

```diff
@@ -266,10 +301,20 @@ def ingest_idx(label_file, table_file, configpath, outputdir):
         outputdir,
         '%s%s' % (instrument, SEGMENT_DB_SUFFIX)
     )
-    segments = store_segments(outputfile, metadata, config)
+    if body_radius == MARS_RADIUS_M:
+        segments = store_segments(outputfile, metadata, config)
+    else:
+        segments = store_segments(outputfile, metadata, config, body_radius)
```

Comment:
Can remove the if/else block and just always pass in body_radius=body_radius argument.

Reply:
removed if/else block

Reply:
updated test/test_ingest.py, to assume called with Mars radius. (https://github.com/JPLMLIA/pdsc/blob/lroc/test/test_ingest.py#L154)

Reply:
added Mars radius to:
 mock_store_segments.assert_called_with(
        os.path.join('test_output', 'instrument_name_segments.db'),
        'metadata', 'config_contents', ### 3396200.0